website/docs/language/backend/remote.mdx

```diff
@@ -7,7 +7,7 @@ description: >-
 
 # remote
 
--> **Note:** We introduced the remote backend in Terraform v0.11.13 and Terraform Enterprise v201809-1. As of Terraform v1.1.0 and Terraform Enterprise v202201-1, **we recommend using the HCP Terraform's built-in [`cloud` integration](/terraform/cli/cloud/settings)** instead of this backend. The `cloud` option includes an improved user experience and more features.
+-> **Note:** We introduced the remote backend in Terraform v0.11.13 and Terraform Enterprise v201809-1. As of Terraform v1.1.0 and Terraform Enterprise v202201-1, **we recommend using the HCP Terraform's built-in [`cloud` integration](/terraform/cli/cloud/settings)** instead of this backend. The `cloud` option includes an improved user experience and more features. For example, Structured Run Output does not work with this backend. 
```

Comment:
Did a little opporunistic clean up to match our style guide and suggestions for brevity:
```suggestion
-> **Note:** We introduced the remote backend in Terraform v0.11.13 and Terraform Enterprise v201809-1. As of Terraform v1.1.0 and Terraform Enterprise v202201-1, we recommend using HCP Terraform's built-in [`cloud` integration](/terraform/cli/cloud/settings) instead. The `cloud` option supports an improved user experience and more features, such as [structured run output mode](/terraform/cloud-docs/workspaces/settings#user-interface), which displays your plan and apply results in a human-readable format.
```